rust-runtime/aws-smithy-async/src/future/fn_stream.rs

```diff
@@ -6,12 +6,14 @@
 //! Utility to drive a stream with an async function and a channel.
 
 use crate::future::rendezvous;
-use futures_util::StreamExt;
```

Comment:
nit: I guess we could mention in the changelog that the type now doesn't implement this or `TryStream`, `TryStreamExt` either, for completion.